tox.ini

```diff
@@ -79,6 +79,7 @@ commands =
 changedir = {toxinidir}
 deps = -rrequirements/docs.txt
 commands =
+    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git@{posargs:main}
```

Comment:
This is an admittedly inconsequential comment, but I'd just remove the `{posargs:main}` from the line:
* If you were to exercise this option by specifying a branch for the core repo, then the next line would croak because the git ref isn't a valid build type for Sphinx.
* In the next line, `posargs` is available so that `tox -e docs linkcheck` is possible.  Conversely, there's no `linkcheck` git ref for the core repo, so line 82 would croak.

However, no one might ever run linkcheck (that'd be sad) and no on might ever try to build the docs against a specific ref of the core repo.  So, the end game is that it's up to you whether you want to revise or not.

Reply:
Ah, good catch. Copy pasta.